skeleton/local-deploy/deployment/README.md

```diff
@@ -0,0 +1,78 @@
+# Deploying Locally
+
+This document is intended for users who wish to deploy their application locally. This was generated from a software template and information regarding image building and deployment are found below. 
+
+## Quick Links
+
+- [Prerequisites](#prerequisite)
+
+- [Building and Pushing Images](#building-and-pushing-image-manually)
+
+- [Deployment](#deploying-to-podman-desktop-ai-lab)
+
+- [Cleanup](#cleanup)
+
+- [Troubleshooting](#troubleshooting)
+
+## Prerequisite
+
+Deploying locally via Podman Desktop is done with the assistance of the `AI Lab Extension`. Please ensure you have the following downloaded and installed:
+- [Podman Desktop 1.8.0+](https://github.com/containers/podman-desktop)
+- [Podman 4.9.0+](https://podman.io/)
+- [Podman Desktop AI Extension](https://github.com/containers/podman-desktop-extension-ai-lab?tab=readme-ov-file#installation)
+
+## Building and Pushing Image Manually
+
+You have chosen `${{ values.imageRegistry }}` as your desired image registry. Please ensure you are logged in and can read/write images from that repository. For example using `quay.io` you can run this command to prompt the entering of credentials:
+```
+podman login quay.io
+```
+
+To rebuild images you can run the following command from the root of the repository:
+```
+bash deployment/build_and_push.sh <absolute path to docker/containerfile>
+```
+
+This script requries you to provide the absolute path to your docker/containerfile as an argument. The script will perform both the building of the image as well as pushing it to the registry location you specified as part of the developer hub setup for this application. You can view the templated values in `build_and_push.sh`.
+
+## Deploying To Podman Desktop AI Lab
```

Comment:
I might suggest moving this to the top, above the manual build steps. Reading through this initially, I thought manually building was required, until I got to this step (and some of my initial review comments may reflect that)